code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -134,18 +134,20 @@
 /datum/species/proc/create_organs(var/mob/living/carbon/human/H) //Handles creation of mob organs.
 
 	for(var/obj/item/organ/organ in H.contents)
-		if((organ in H.organs) || (organ in H.internal_organs))
+		if((organ in H.organs))
```

Comment:
This will only look for external organs, as limbs are stored in the H.organs list.